test/metabase/driver/sql/query_processor_test.clj

```diff
@@ -265,7 +291,7 @@
                                :alias        "u"
                                :condition    [:= $user_id &u.users.id]}]}))))))
 
-(deftest joined-field-clauses-test
+(deftest ^:parallel joined-field-clauses-test
```

Comment:
I couldn't help marking a bunch of tests `^:parallel`. Make everything parallel 🌟